web.config

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <system.webServer>
+    <httpErrors errorMode="Detailed" />
+    <rewrite>
+      <rules>
+        <!--BEGIN SSL-->
+        <rule name="ForceSSL" stopProcessing="true">
+          <match url="(.*)" />
+          <conditions>
+            <add input="{HTTPS}" pattern="^OFF$" ignoreCase="true" />
+          </conditions>
+          <action type="Redirect" url="https://{HTTP_HOST}/{R:1}" redirectType="Permanent" />
+        </rule>
+        <!--END SSL-->
+        <!--BEGIN Trailing slash enforcement-->
+        <rule name="Add trailing slash" stopProcessing="true">
+          <match url="(.*[^/])$" />
+          <conditions>
+            <add input="{REQUEST_FILENAME}" matchType="IsFile" negate="true" />
+            <add input="{REQUEST_FILENAME}" matchType="IsDirectory" negate="true" />
+            <add input="{REQUEST_FILENAME}" pattern="(.*?)\.[a-zA-Z]{1,4}$" negate="true" />
+          </conditions>
+          <action type="Redirect" redirectType="Permanent" url="{R:1}/" />
+        </rule> 
+        <!--END Trailing slash enforcement-->
+        <!--BEGIN Primer React Components-->
+        <rule name="React proxy" stopProcessing="true">
+          <match url="^react/(.*)" ignoreCase="true" />
```

Comment:
Do we also need to match the URL /components here? I noticed on the preview site https://primerstyle.azurewebsites.net/ that the link to React Components on the homepage would 404: https://primerstyle.azurewebsites.net/components/

Reply:
yes, `/components` is just a 301 redirect to `/react`. It's what PRC used to be called before we renamed it to just `react`. If we wanted to keep the redirect, I'll need to add a separate rule in. I'll give it a go 👍 .